app.dstack.yaml

```diff
@@ -0,0 +1,50 @@
+type: service
+
+name: aana-chat-with-video
+
+image: nvidia/cuda:12.3.2-cudnn9-devel-ubuntu22.04
+
+env:
+  - NUM_WORKERS=5
+  - TASK_QUEUE__EXECUTION_TIMEOUT=10000
+  - TASK_QUEUE__NUM_WORKERS=5
+  - TMP_DATA_DIR=/demo_data/aana
+  - IMAGE_DIR=/demo_data/aana/images
+  - VIDEO_DIR=/demo_data/aana/videos
+  - AUDIO_DIR=/demo_data/aana/audios
+  - MODEL_DIR=/demo_data/aana/models
+  - DB_CONFIG__DATASTORE_TYPE=sqlite
+  - DB_CONFIG__DATASTORE_CONFIG__PATH=/demo_data/aana.db
+
+commands:
+  - apt-get update
+  - apt-get install -y libgl1 libglib2.0-0 ffmpeg python3 python3-dev git nvtop htop sqlite3 cron
+  - curl -sSL https://install.python-poetry.org | python3 -
+  - export PATH=$PATH:/root/.local/bin
+  - sh install.sh
+  - mkdir -p /demo_data
+  - mkdir -p /demo_data/hf_cache
+  - (crontab -l 2>/dev/null; echo "0 3 * * * rm -f $TMP_DATA_DIR/videos/*") | sort -u | crontab -
+  - (crontab -l 2>/dev/null; echo "0 3 * * * rm -f $TMP_DATA_DIR/audios/*") | sort -u | crontab -
+  - (crontab -l 2>/dev/null; echo "0 3 * * * rm -f $TMP_DATA_DIR/images/*") | sort -u | crontab -
```

Comment:
These cronjobs should have a time check on the files
for example delete all the files that are older than 3 days!!
`crontab -l 2>/dev/null; echo "0 3 * * * find $TMP_DATA_DIR/images/* -type f -mtime +3 -exec rm -f {} \;" | sort -u | crontab -`

Reply:
Good idea! I will add it.

Reply:
Done. I changed it to check access time because mtime is not correct for the videos, it probably has something to do with yl-dlp.